### ISSUE TITLE
Translation issues

### ISSUE THREAD

**Issue by HarunKuru** [dnGrep/dnGrep#1178]
I translated dnGrep to Turkish and encountered the following problems:

Problem 1:

label:"Automatically translated" problem:
Main_MoreArrowButton
Description: It appears as "Labels: Automatically translated". Since it is the same in the original file, it cannot be changed.

Problem 2:

Keys that give "has:dismissed-check" check error:
Options_CopyFiles
Description: "Copy files" translation gives "Reused translation" error.

ReportSummary_Multiline
Description: "Multiline" translation gives "Reused translation" error.

Problem 3:

dnGrep interface fields that cannot be translated in the program even though the translation file is translated:

Main_Menu_NewScript
Description: It continues to appear as "New Script...".

Main_Menu_EditScript
Description: It continues to appear as "Edit Script..."

Main_Menu_CancelScript
Description: It continues to appear as "Cancel Script".

From the child elements of the Options_PersonalizeControlVisibility key:

Options_Personalize_Features
Description: It continues to appear as "Features".

Options_Personalize_FileFilter
Description: It continues to appear as "File filter".

Options_Personalize_SpecialOptions
Description: It continues to appear as "Special options".

Options_Personalize_SearchType
Description: It continues to appear as "Search type".

Options_Personalize_SpecialOptions
Description: It continues to appear as "Special options".

Options_Personalize_ResultOptions
Description: Still appears as "Result options".

Options_Personalize_ResultsTree
Description: Still appears as "Results tree".

Options_Personalize_PreviewWindow
Description: Still appears as "Preview window".

**Comment by doug24** [comment]
Thank-you for doing this!

I merged your changes from Weblate, and can run them in the UI, which is a little different than the experience you get loading the file manually.

Problem 1:  the Main_MoreArrowButton appears to be translated correctly in both Weblate and in dnGrep - maybe this isn't a problem?

Problem 2: I don't know what that message is for, or how to get it to go away.  But I think the translations are ok?

Problem 3: The Script menu: this is a bug in dnGrep, it is not recreating those dynamic menus when the language is changed.  It is only noticeable when first changing the language, if you restart with Turkish saved, they will be in Turkish.  But it is a bug, and I'll fix it.

The personalization group labels: this is also a bug, they are not getting recreated when the translation changes.  If you close and reopen the Options dialog you will see your translated labels.

Problem 4: I had not seen the red border highlight before, thanks for pointing that out.  Also a bug and an easy fix.

**Comment by HarunKuru** [comment]
Hi Doug P...

First of all, thank you for your answer and for such a great program.

When I checked the Weblate language file again, I saw that it no longer gave any errors.

I don't know if it's because it's been approved, but the errors I mentioned in my previous message are no longer visible. I think that since the next update regarding the dnGrep interface will include a Turkish language pack, those problems will also be eliminated.

Apart from these, there are some minor errors I found in the program. And of course, I have some opinions and suggestions. Should I open a new topic or should I continue from here?
